src/prefect/agent/ecs/agent.py

```diff
@@ -224,14 +232,20 @@ def __init__(  # type: ignore
         else:
             self.run_task_kwargs = {}
 
-        # If running on fargate, auto-configure `networkConfiguration` for the
-        # user if they didn't configure it themselves.
-        if self.launch_type == "FARGATE" and not self.run_task_kwargs.get(
-            "networkConfiguration"
-        ):
-            self.run_task_kwargs[
-                "networkConfiguration"
-            ] = self.infer_network_configuration()
+        if not self.run_task_kwargs.get("capacityProviderStrategy"):
+            self.launch_type = launch_type.upper() if launch_type else "FARGATE"
+
+            self.logger.error(
```

Comment:
Is error level logging too high? Because it will still run since it's handled by the logic here right?